stylesheets/_global.scss

```
@@ -714,6 +714,14 @@ $loading-height: 16px;
   .button {
     background: $color-loki-green-gradient;
     border-radius: 100px;
+
+    &[disabled='disabled'] {
```

```suggestion
    &:disabled, &:disabled:hover {
```

I think these may also work and make it look cleaner